app/hooks/use-logout.ts

```diff
@@ -0,0 +1,17 @@
+import { useApolloClient } from "@apollo/client"
+import { resetDataStore } from "../utils/logout"
+import useToken from "../utils/use-token"
+
+const useLogout = () => {
+  const client = useApolloClient()
+  const { removeToken } = useToken()
+
+  // TODO a login function could be created in this hook
+  const logout = () => resetDataStore({ client, removeToken })
```

Comment:
We could bring in the navigator using the [useNavigation hook](https://reactnavigation.org/docs/5.x/use-navigation) and always navigate the user to the Home Screen on logout.

Reply:
I tried but there is on problem to do this, because `GlobalError` is not under any navigator so you can't redirect to any screen from GlobalError. 
**BUT** anyways this code in `RootNavigator` is redirecting to getStarted:
```
 <RootNavigator.Navigator
      screenOptions={{ gestureEnabled: false }}
      initialRouteName={token ? "authenticationCheck" : "getStarted"}
    >
```